platform-controller/src/main/java/org/hobbit/controller/docker/ClusterManager.java

```diff
@@ -25,6 +25,15 @@
      */
     public Integer getNumberOfNodes() throws DockerException, InterruptedException;
```

Comment:
I would suggest the general usage of `int` instead of `Integer` since the latter creates more effort for the JVM. Although this won't influence the overall performance it is better to "train" this behavior :wink: